src/components/TipInput.vue

```diff
@@ -136,12 +137,12 @@ export default {
           token: null,
         }
         : {
-          value: this.largestFtTipAmount.amount,
-          token: this.largestFtTipAmount.token,
+          value: this.largestFtTipAmount ? this.largestFtTipAmount.amount : null,
+          token: this.largestFtTipAmount ? this.largestFtTipAmount.token : null,
         };
     },
     v1TipWarning() {
-      return this.tip && this.tip.id.split('_')[1] === 'v1' && this.inputToken !== 'native';
+      return this.tip && this.tip.id.split('_')[1] === 'v1' && this.inputToken !== null;
```

Comment:
```suggestion
      return this.tip && this.tip.id.split('_')[1] === 'v1' && this.inputToken;
```
Is there any reason to explicitly compare inputToken with null?

Reply:
Probably it would have sense if we use TypeScript.

Reply:
explicit null means native